docs/using/administering/debugging.rst

```diff
@@ -119,6 +119,47 @@ Alternatively, the information can be gathered manually using the following comm
 
 * Flocker log files (see :ref:`Flocker logging <flocker-logging>` above)
 
+Profiling
+---------
+
+.. warning::
+
+   It is not recommended to use profiling while relying on Flocker within a production environment as there is a performance overhead.
+
+Control Service
+^^^^^^^^^^^^^^^
+
+It is possible to obtain :py:mod:`cProfile` profiling data of the :ref:`Control Service <control-service>` between two user defined intervals.
+
+Profiling is disabled by default.
+To enable profiling of the Control Service run the following command as root on the control node:
+
+.. prompt:: bash #
+
+   pkill -SIGUSR1 flocker-control
+
+Profiling data will then be collected until a signal to dump the profiling data is received.
+To dump the current profile run the following command as root on the control node:
+
+.. prompt:: bash #
+
+   pkill -SIGUSR2 flocker-control
+
+This will output the profiling data to a file named :file:`/etc/flocker/profile-<TIMESTAMP>`.
+This will also disable profiling.
+When profiling is re-enabled, new profiling data will be added to the existing profiling data until the profile is dumped again.
```

Comment:
This line is a bit confusing. Do you mean something like: "If profiling is re-enabled in the same run of the control server, new profiling data will be added to all previously collected profiling data when the profile is dumped again"